src/client/internal.ts

```diff
@@ -217,6 +221,14 @@ export interface RemoveIntentListenerPayload {
     intent: string;
 }
 
+export interface addContextListenerPayload {
```

Comment:
There is precedent for empty payload interfaces earlier in the file. Figured this would make things easier if we ever need to add any new properties here.